py-polars/tests/unit/test_consortium_standard.py

```diff
@@ -0,0 +1,35 @@
+"""
+Test some basic methods of the dataframe consortium standard.
+
+Full testing is done at https://github.com/data-apis/dataframe-api-compat,
+this is just to check that the entry point works as expected.
+"""
+import pytest
+
+import polars as pl
+
+pytest.importorskip("dataframe_api_compat")
```

Comment:
We shouldn't be skipping tests like this. The functionality is in the library, so it must be tested. You'll just have to make sure your dev environment is up to date.